### ISSUE TITLE
Head light not visible of other players in multiplayer

### ISSUE THREAD

**Issue by CyaNox** [tsunamayo/Skywanderers#229]
The headlight of other players is not visible nor is the sound played for other players when toggling it on and off repeatedly.

**Comment by tsunamayo** [comment]
Yes correct it is yet to be implemented, multi is very much wip - but please dont hesitate to report anything concerning mp. It will be fixed for the lighting refactoring - when the game will switch in deferred mode. Thanks!
